src/hssm/hssm.py

```diff
@@ -537,7 +539,8 @@ def sample_posterior_predictive(
         inplace: bool = True,
         include_group_specific: bool = True,
         kind: Literal["pps", "mean"] = "pps",
-        n_samples: int | float | None = None,
+        draws: int | float | list[int] | None = None,
+        safe_mode: bool = True,
```

Comment:
I suggest adding `safe_mode` in the function documentation.

Reply:
Adressed